install.sh

```diff
@@ -1,96 +1,143 @@
 #!/usr/bin/env bash
 set -e
 
-echo Installing protostar
-
-PROTOSTAR_DIR=${PROTOSTAR_DIR-"$HOME/.protostar"}
-mkdir -p "$PROTOSTAR_DIR"
-
-
-PLATFORM="$(uname -s)"
-case $PLATFORM in
-  Linux)
-    PLATFORM="Linux"
-    ;;
-  Darwin)
-    PLATFORM="macOS"
-    ;;
-  *)
-    echo "unsupported platform: $PLATFORM"
-    exit 1
-    ;;
-esac
+PROTOSTAR_REPO="https://github.com/software-mansion/protostar"
+RESULT=""
```

Comment:
1. I have seen name `RETVAL` in the wild (for example here: https://github.com/rust-lang/rustup/blob/master/rustup-init.sh)
2. This statement is NOOP as far as I know bash. You can safely remove it.

Reply:
Removed global declaration. Renamed to retval